src/clj/puppetlabs/services/jruby/jruby_puppet_service.clj

```diff
@@ -15,18 +16,25 @@
 
 (trapperkeeper/defservice jruby-puppet-pooled-service
                           jruby/JRubyPuppetService
-                          [[:ConfigService get-config]
-                           [:ShutdownService shutdown-on-error]
-                           [:PuppetProfilerService get-profiler]
-                           [:PoolManagerService create-pool]]
+                          {:required [[:ConfigService get-config]
+                                      [:ShutdownService shutdown-on-error]
+                                      [:PuppetProfilerService get-profiler]
+                                      [:PoolManagerService create-pool]]
+                           :optional [MetricsService]}
```

Comment:
@rlinehan As we talked about IRL yesterday, I'm good with the idea of creating a new setting which would allow a user to turn off the `http-client` specific metrics in the event that they may cause performance problems, etc.  Where do you think such a setting should live?  I was thinking maybe having a `metrics-enabled` setting in the `http-client` area of the TK config, where we already have client-specific settings like `ssl-protocols` and `cipher-suites`, might make sense.  Could default the setting if not specified to `true` just to make it easier for users to use them for the majority cases where hopefully they won't cause any problems.  WDYT?